packages/sui-mono/src/spawn.js

```diff
@@ -0,0 +1,55 @@
+/* eslint no-console:0 */
```

Comment:
I created it at first for sui-precommit and copied it there to have a single way to execute...
command execute have multiple ways all over the packages. Most of them trim colors and newlines of the output. But in this case execute does about the same thing.
Anyways serialSpawn was about to be moved to sui-helpers to execute commands from there and have a single way to to it.